### PR TITLE
Certs in metadata "shoulds"

### PR DESCRIPTION
**Pull request by ergood**
In response to issue #80. 

Created non-normative text to outline SHOULD/SHOULD NOT advice for interop with non-compliant implementations.

### CODE REVIEW

**Comment by scantor** [comment]
Similar to my other comment about avoiding capitalized keywords in italics, I'd reword around something like "the following are suggested practices to avoid interoperability issues with deployments outside the scope of this profile" or something similar.

**Comment by ergood** [comment]
Updated to remove the all caps/normative format in the non-normative text.
